frontend/www/js/omegaup/components/arena/RunDetails.vue

```diff
@@ -297,5 +357,21 @@ export default class ArenaRunDetails extends Vue {
     const visible = this.groupVisible[group];
     this.$set(this.groupVisible, group, !visible);
   }
+
+  showDataCase(
+    cases: types.ProblemCasesContents,
+    caseName: string,
+    caseType: string,
+  ): string {
+    switch (caseType) {
+      case 'in':
+        return cases[caseName]?.in ?? EMPTY_FIELD;
+      case 'out':
+        return cases[caseName]?.out ?? EMPTY_FIELD;
+      case 'contestantOutput':
+        return cases[caseName]?.contestantOutput ?? EMPTY_FIELD;
+    }
+    return EMPTY_FIELD;
+  }
```

Comment:
esto me funcionó a mí:

```suggestion
  showDataCase(
    cases: types.ProblemCasesContents,
    caseName: string,
    caseType: 'in'|'out'|'contestantOutput',
  ): string {
    return cases[caseName]?.[caseType] ?? EMPTY_FIELD;
  }
```

Reply:
Ese truco no me lo sabía. Gracias.